osc/commandline.py

```diff
@@ -2663,9 +2642,8 @@ def _set_bugowner(self, args, opts):
 
         return actionxml
 
-# TODO: fix ValueError: unknown action "callback"
-#    @cmdln.option('-a', '--action', action='callback', callback = _actionparser, dest = 'actions',
-#                  help='specify action type of a request, can be : submit/delete/change_devel/add_role/set_bugowner')
+    @cmdln.option('-a', '--action', action='append', nargs='+', metavar=('ACTION', '[ARGS]'), dest = 'actions',
```

Comment:
Remove spaces from `dest = 'actions'` -> `dest='actions'` (nit-picking, but since you're touching the line...)
Add `default=[]` otherwise `osc creq` fails with a traceback.